server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+
+_bearer-only_::
+  Bearer-only access type means that the application only allows bearer token requests.
+  If this is turned on, this application cannot participate in browser logins.
+
+*Standard Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Authorization Code Flow>>.
+
+*Implicit Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Implicit Flow>>.
+
+*Direct Access Grants Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Direct Access Grants>>.
+
+*Root URL*
+
+If {project_name} uses any configured relative URLs, this value is prepended to them.
+
+*Valid Redirect URIs*
+
+This is a required field.  Enter in a URL pattern and click the + sign to add.  Click the - sign next to URLs you want to remove.
+Remember that you still have to click the `Save` button!
+Wildcards (*) are only allowed at the end of a URI, i.e. $$http://host.com/*$$
+
+You should take extra precautions when registering valid redirect URI patterns. If you make
+them too general you are vulnerable to attacks.  See <<_unspecific-redirect-uris, Threat Model Mitigation>> chapter
+for more information.
+
+*Base URL*
+
+If {project_name} needs to link to the client, this URL is used.
+
+*Admin URL*
+
+For {project_name} specific client adapters, this is the callback endpoint for the client.  The {project_name}
+server will use this URI to make callbacks like pushing revocation policies, performing backchannel logout, and other
+administrative operations.  For {project_name} servlet adapters, this can be the root URL of the servlet application.
+For more information see link:{adapterguide_link}[{adapterguide_name}].
+
+*Web Origins*
+
+This option centers around link:https://fetch.spec.whatwg.org/[CORS] which stands for Cross-Origin Resource Sharing.
+If browser JavaScript tries to make an AJAX HTTP request to a server whose domain is different from the one the
+JavaScript code came from, then the request must use CORS.
+The server must handle CORS requests in a special way, otherwise the browser will not display or allow the request to be processed.
+This protocol exists to protect against XSS, CSRF and other JavaScript-based attacks.
+
+{project_name} has support for validated CORS requests.  The way it works is that the domains listed in the
+`Web Origins` setting for the client are embedded within the access token sent to the client application.  The client
+application can then use this information to decide whether or not to allow a CORS request to be invoked on it.  This is
+an extension to the OIDC protocol so only {project_name} client adapters support this feature.
+See link:{adapterguide_link}[{adapterguide_name}] for more information.
+
+To fill in the `Web Origins` data, enter in a base URL and click the + sign to add.  Click the - sign next to URLs you want to remove.
```

Comment:
"To add "Web Origins" data, enter the base URL and click +. Click - to remove a URL."